set/string_test.go

```diff
@@ -0,0 +1,180 @@
+package set_test
+
+import (
+	"testing"
+
+	"github.com/reddit/baseplate.go/set"
+)
+
+func TestStringSet(t *testing.T) {
+	const (
+		key1 = "key1"
+		key2 = "key2"
+	)
+
+	var s set.String
+
+	t.Run(
```

Comment:
Is that an alternative to table-driven testing to make debugging of tests easier?

Reply:
This is more like control the order of sequential (sub-)tests, and also share state between them.